app/CsvImportModule/Contracts/CsvDataImporter.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+
+namespace App\CsvImportModule;
+
+
+interface CsvDataImporter
+{
+    /**
+     * @param CsvExtractedData $csvExtractedData
+     * @param string $entityClassName
+     * @param CsvColumnMapper|null $columnMapper
+     *
+     */
+    static function import(CsvExtractedData $csvExtractedData, string $entityClassName, CsvColumnMapper $columnMapper): void;
```

Comment:
If you declare argument as null in function doc, I believe it makes sense to add it a default value to null in signature as null, like `CsvColumnMapper $columnMapper = null`